core-blocks/gallery/index.js

```diff
@@ -125,12 +125,15 @@ export const settings = {
 				},
 			},
 			{
+				// When created by drag and dropping multiple files on an insertion point
 				type: 'files',
 				isMatch( files ) {
 					return files.length !== 1 && every( files, ( file ) => file.type.indexOf( 'image/' ) === 0 );
 				},
 				transform( files, onChange ) {
-					const block = createBlock( 'core/gallery' );
+					const block = createBlock( 'core/gallery', {
```

Comment:
Actually, I noticed that `Image` block uses `createBlobURL` from `@wordpress/blob` package. Can we use it instead? It has additional cache layer.

Reply:
im on it!

Reply:
hey @gziolo would you mind reviewing this https://github.com/WordPress/gutenberg/pull/7734 ?